File: read_equipment.js
import XLSX from 'xlsx';
import { readFileSync } from 'fs';

const workbook = XLSX.read(readFileSync('attached_assets/EQUIPMENT MASTER LIST -SEPTEMBER 2018 E.C_1759791262397.xlsx'));
const sheetName = workbook.SheetNames[0];
const worksheet = workbook.Sheets[sheetName];
const data = XLSX.utils.sheet_to_json(worksheet);

console.log(JSON.stringify(data, null, 2));

File: read_equipment.py
import pandas as pd
import json

# Read the Excel file
df = pd.read_excel('attached_assets/EQUIPMENT MASTER LIST -SEPTEMBER 2018 E.C_1759791262397.xlsx')

# Convert to JSON
print(json.dumps(df.to_dict(orient='records'), indent=2, default=str))
